benchbuild/utils/settings.py

```diff
@@ -281,8 +280,8 @@ def init_from_env(self):
                 env_val = self.node['default']
             env_val = os.getenv(env_var, to_yaml(env_val))
             try:
-                self.node['value'] = yaml.load(
-                    str(env_val), Loader=ConfigLoader)
+                self.node['value'] = yaml.load(str(env_val),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of unsafe yaml load. Allows instantiation of arbitrary objects. Consider yaml.safe_load().](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)